src/scss/_variables.scss

```diff
@@ -0,0 +1 @@
+$primary: rgb(13, 143, 13);
```

Comment:
This is the absolute bare minimum in terms of Bootstrap customisation. I think you can do more!

Reply:
more added